### ISSUE TITLE
No clipboard support

### ISSUE THREAD

**Issue by littlephil** [mpcjanssen/simpletask-android#884]
I am using an OPPO F1 and am unable to select any text in the app, copy/cut or paste from other apps.
I have experimented with the standard long press and double tapping the screen but nothing seems to work.

**Comment by smichel17** [comment]
Huh, works fine for me. Is this only for the AddTask screen or any of the text boxes in the app (ex: the dialog for changing lists/tags)?

**Comment by mpcjanssen** [comment]
This is probably from the main screen, which doesn't support copy/paste. You can select tasks to copy -> edit -> copy/paste from there or (what I do) select the tasks to copy. And share them to clipboard.

**Comment by littlephil** [comment]
The problem occurs when trying to select, copy or paste in any text field in the application by using a long press. I am not trying to copy whole tasks.
  
I have tried changing my keyboard in case this is causing the problem but the result was the same.
  

  
One use case would be:
  
I copy a URL from my browser into my clipboard.
  
I create a new task in Simpletask using the + button.
  
I press and hold on the text area.
  

  
Expected behaviour:
  
A context menu appears with the paste option.
  

  
Actual behaviour:
  
No context menu is displayed.
  
    
  
  
>

**Comment by mpcjanssen** [comment]
This works fine for me, so it's difficult to fix if I can't reproduce it. It might be related to the specific EditText settings we are using though. 
As a workaround note that instead of copying text from the original place, you can also share it with Simpletask.

**Comment by littlephil** [comment]
Okay, good workaround for now. If there's anything that I can do to help regarding logs/testing etc. I'd be very happy to help.
  
  
  

  
  
  
  
  
>

**Comment by mpcjanssen** [comment]
textIsSelectable might be the magic property.

**Comment by mpcjanssen** [comment]
@littlephil do you use the cloudless or dropbox version? And the one from f-droid or the playstore?

**Comment by littlephil** [comment]
I use the one from the Playstore, it's the one with Nextcloud integration built in. It's shows as "Simpletask N" on my phone.   

  

  
  
  
  
  
>

**Comment by mpcjanssen** [comment]
@littlephil can you try with the nextcloud release apk from https://cloud.mpcjanssen.nl/index.php/s/9QtrYDN2z7Kwtko ?

**Comment by littlephil** [comment]
No joy, same result.

I've also noticed the spelling correction doesn't function in any text box either. If that helps :)

**Comment by mpcjanssen** [comment]
@littlephil could you try again with the apk from the previous URL?

**Comment by littlephil** [comment]
Tried app-nextcloud-release but no change.
  
  
  

  
  
  
  
  
>

**Comment by mpcjanssen** [comment]
@littlephil very strange. The latest version uses a standard edittext without any modifications. Can you copy paste in other applications?

**Comment by littlephil** [comment]
Yes, pretty much all of them. I have found applications like an email app where I can't copy/paste in the subject but can in the body. It is strange. Curious if the same happens on other OPPOs, I will try to get my friend to try there's when I'm in Delhi in May.
  
  
  

  
  
  
  
  
>

**Comment by smichel17** [comment]
@mpcjanssen maybe we could send @littlephil a debug apk where the text box is completely standard, to see if one of our modifications is breaking clipboard support or whether it's a problem with android (that we can't easily fix)?

**Comment by mpcjanssen** [comment]
@smichel17 that is exactly what my last build was.

**Comment by mpcjanssen** [comment]
Only thing I can imagine is that this was related to the empty text in the textbox (which also caused the  phone number completion).
@littlephil is this still an issue in the last version from the playstore (10.2.7 as of now)?

**Comment by littlephil** [comment]
Yes, updated and still no reaction to a long press to paste text or even to select.
  
  
  

  
  
  
  
  
>
